### PR TITLE
topology: soundwire: cleanup -rt700 files and add support for RTK 3-in-1 board

### PR DESCRIPTION
**Pull request by plbossart**
These patches could also be applied to @slawblauciak 's https://github.com/thesofproject/sof/pull/1638

with the latest kernel branch for SoundWire playback and capture seem to be functional with the 3-in-1 board (except for the timeout issue in MSI mode).

### CODE REVIEW

File: tools/topology/sof-icl-rt711-rt1308-rt715.m4 [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+
+# Include TLV library
+include(`common/tlv.m4')
+
+# Include Token library
+include(`sof/tokens.m4')
+
+# Include ICL DSP configuration
+include(`platform/intel/icl.m4')
+include(`platform/intel/dmic.m4')
```
Comment (bardliao) [inline tools/topology/sof-icl-rt711-rt1308-rt715.m4]: We may not need dmic.m4 since there is no dmic in this file now.

Comment (plbossart) [reply tools/topology/sof-icl-rt711-rt1308-rt715.m4]: good catch, will fix. thanks!

Comment (plbossart) [reply tools/topology/sof-icl-rt711-rt1308-rt715.m4]: updated now

**Comment by plbossart** [comment]
@slawblauciak you may want to take the first commit since you've done a set of squashes in the official ALH PR. The last two are useful as well, may be useful to have them all in one PR.

**Comment by jajanusz** [comment]
@plbossart please rebase, CI cannot pass cos it cannot build TGL

**Comment by plbossart** [comment]
Changes merged in #1638, closing this PR

**Pull request closed without merging**